src/ert/config/field.py

```diff
@@ -95,6 +95,14 @@ class Field(ParameterConfig):
     grid_file: str
```

Comment:
Not sure about grid file. Does it always need to be specified - it is either the global one or as a named attribute for FIELD - this is actually WIP though.